lib/taskana-core/src/main/java/pro/taskana/common/internal/jobs/JobRunner.java

```diff
@@ -114,6 +149,24 @@ private void runScheduledJob(ScheduledJob scheduledJob) {
               + e.getMessage(),
           e);
     }
-    LOGGER.debug("exit from runScheduledJob");
+  }
+
+  private Subject getAdminSubject() {
```

Comment:
We do have TaskanaEngineImpl.runAsAdmin(). Do you think we could reuse this here instead of creating a separate method?

Reply:
This was my first idea: use InternalTaskanaEngineImpl.runAsAdmin. However, that method is a method on InternalTaskanaEngineImpl and JobRunner  has no access to InternalTaskanaEngineImpl, only to TaskanaEngineImpl.